python/lsst/ts/xml/data/sal_interfaces/LEDProjector/LEDProjector_Events.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <Enumeration>
+    LEDBasicState_Unknown=0,
+    LEDBasicState_Off=1,
+    LEDBasicState_On=2,
+    LEDBasicState_UnexpectedlyOn=3,
```

Comment:
What does UnexpectedlyOn/Off mean? What would you see to report that state?

Reply:
I kept it from ATWhiteLight. I suspect when there is a mismatch of states where the Python thinks we are on, or off, and the labjack reports that we are actually in the opposite state. I'm not sure if its needed but I figured better to have it and not need it and we can remove later.

Reply:
Makes sense.